### PR TITLE
Change editors to current team members

### PR DESCRIPTION
**Pull request by arnau**
Former editors are no longer part of the Registers team or GDS.

## Review

Check names are correctly spelled.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
```
Comment (gidsg) [inline Dockerfile]: Did you mean to add this file in? There's nothing 'bout it in the PR description.

Comment (arnau) [reply Dockerfile]: I doubted if opening a new PR would be better but ended up going with all in. I can split it if it makes more sense.

Comment (arnau) [reply Dockerfile]: I've opened another PR for docker #69

File: index.html [hunk 2]
```diff
@@ -34,6 +34,7 @@
  *   - .note       for informative notes             (div, p, span, aside, details)
  *   - .example    for informative examples          (div, p, pre, span)
  *   - .issue      for issues                        (div, p, span)
+ *   - .assertion  for assertions                    (div, p, span)
```
Comment (gidsg) [inline index.html:37]: it's hard to follow all these changes. Have we changed the generator at the same time and it has resulted in all these changes?

Comment (arnau) [reply index.html:37]: You should ignore the index.html diff as it is the reflection of index.bs + dependencies. Same reasoning as I said in #69

**Review by michaelabenyohai — APPROVED** [review]
